jscomp/bsb/templates/react/src/simple/page.re

```diff
@@ -0,0 +1,13 @@
+let component = ReasonReact.statefulComponent "Greeting";
+
+let make ::name _children => {
+  let click _event state _self => ReasonReact.Update (state + 1);
+  {
+    ...component,
+    initialState: fun () => 0,
+    render: fun state self => {
+      let greeting = {j|Hello $name, You've clicked the button $state times(s)!|j};
```

Comment:
@chenglou A very small piece of first time user experience feedback :) I got slightly confused when I was playing with the `simple` template, went to `simpleRoot.re` and pass `2` instead of `Lil' Reason` as the `name` prop when I saw I wasn't getting any errors. "It should infer that `name` is a string!", I thought.

I don't understand the internals, but I guess because of the string interpolation being done here, the type inference concludes `name` is of type `'a` and thus anything can be passed to it, right?

Would it make sense to annotate `name`? Or use `String.concat` here?

Reply:
Funny coincidence, I just submitted #1981, which fixes it =).

Thanks for the tip!